Core/Configuration/elxConfiguration.cxx

```diff
@@ -18,9 +18,49 @@
 
 #include "elxConfiguration.h"
 
+#include "elxConversion.h"
+#include "elxTransformIO.h"
+
 namespace elastix
 {
+namespace
+{
+itk::ParameterFileParser::ParameterMapType
+AddDataFromExternalTransformFile(itk::ParameterFileParser::ParameterMapType parameterMap)
```

Comment:
@mstaring FYI Based on your feedback, I just looked at the possibility of passing the `parameterMap` parameter by reference, and then doing the adjustments "in-place". This does not appear possible, because the original parameter map argument in both `Configuration::Initialize` overloads is a `const` reference! So I think it's best to just leave it this way, for now.